src/navigation-bar.tsx

```diff
@@ -0,0 +1,524 @@
+            </div>
+        </div>
+    );
+};
+
+const NAVBAR_ZINDEX = 25;
+const BURGER_ZINDEX = 26;
+
+const DESKTOP_NAVBAR_HEIGHT = 80;
+const MOBILE_NAVBAR_HEIGHT = 56;
+
+const BURGER_MENU_ANIMATION_DURATION_MS = 300;
+
+const useStyles = createUseStyles((theme) => {
+    const shadowAlpha = theme.isDarkMode ? 1 : 0.2;
+
+    return {
+        topFixed: {
+            position: 'fixed',
+            top: 0,
+            left: 0,
+            right: 0,
+            zIndex: NAVBAR_ZINDEX,
+        },
+        navbar: {
+            width: '100%',
+            display: 'flex',
+            alignItems: 'center',
+            background: ({isInverse}) =>
+                isInverse ? theme.colors.navigationBarBackground : theme.colors.background,
+            height: DESKTOP_NAVBAR_HEIGHT,
+            padding: '16px 0',
+            transition: 'border 300ms',
+            borderBottom: `1px solid ${theme.colors.divider}`,
+            borderColor: ({isInverse}) =>
+                isInverse && !theme.isDarkMode ? 'transparent' : theme.colors.divider,
+            [theme.mq.tabletOrSmaller]: {
+                borderColor: ({isMenuOpen, isInverse}) =>
+                    isMenuOpen || (isInverse && !theme.isDarkMode) ? 'transparent' : theme.colors.divider,
+                height: MOBILE_NAVBAR_HEIGHT,
+                padding: '8px 0',
+            },
+        },
+        section: {
+            height: DESKTOP_NAVBAR_HEIGHT,
+            display: 'flex',
+            alignItems: 'center',
+            padding: '0 8px',
+            borderBottom: `2px solid transparent`,
+            transition: 'border-color 300ms ease-in-out',
+        },
+        selectedSection: {
+            borderColor: ({isInverse}) => (isInverse ? theme.colors.inverse : theme.colors.controlActivated),
+        },
+        spacer: {
+            height: DESKTOP_NAVBAR_HEIGHT,
+            [theme.mq.tabletOrSmaller]: {
+                height: MOBILE_NAVBAR_HEIGHT,
+            },
+        },
+        burgerMenu: {
+            zIndex: BURGER_ZINDEX,
+            position: 'fixed',
+            top: MOBILE_NAVBAR_HEIGHT,
+            left: 0,
+            right: 0,
+            height: `calc(100vh - ${MOBILE_NAVBAR_HEIGHT}px)`,
+            overflowY: 'auto',
+            background: theme.colors.background,
+            boxShadow: ({menuTransitionState}) =>
+                menuTransitionState !== 'closed' ? `6px 0 4px -4px rgba(0, 0, 0, ${shadowAlpha})` : 'none',
+            transition: `transform ${BURGER_MENU_ANIMATION_DURATION_MS}ms ease-out`,
+        },
+        burgerMenuEnter: {
+            transform: 'translate(-100vw)',
+        },
+        burgerMenuEnterActive: {
+            transform: 'translate(0)',
+        },
+        burgerMenuExit: {
+            transform: 'translate(0)',
+        },
+        burgerMenuExitActive: {
+            transform: 'translate(-100vw)',
+        },
+    };
+});
+
+type MainNavigationBarSection =
```

Comment:
Missing hover state in desktop views
https://zpl.io/29dLjWp

Reply:
done ;)